### PR TITLE
Fixed Scheduler: "task failed" counts twice

### PR DESCRIPTION
**Pull request by Shubhamverma2796**
Pull Request for Issue #36859 .

### Summary of Changes
Removed +1 from the 'time_failed' query

### Testing Instructions

1. Go to Scheduled Tasks and add a GET Request
2. Enter an invalid URL to Request URL (e.g. http://jooomla.org)
3. Execution Rule: manual
4. Save the task
5. Run the task once

### Actual result BEFORE applying this Pull Request
151224355-56e84688-30d2-4ce4-85f8-0bf100bb3051

The count of 'Times failed' is incorrect. It should be equal to the Times Executed.

### Expected result AFTER applying this Pull Request

The count of 'Times failed' increments by +1 only if the task fails.
151224512-a0729bd7-57ac-4bf1-8a73-1ff53faab40d

### CODE REVIEW

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 78037c2ec72e4ab36a119b6dead08973bd93494d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36868">issues.joomla.org/tracker/joomla-cms/36868</a>.</sub>

**Comment by pritam825** [comment]
I have tested this item :white_check_mark: successfully on 78037c2ec72e4ab36a119b6dead08973bd93494d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36868">issues.joomla.org/tracker/joomla-cms/36868</a>.</sub>

File: administrator/components/com_scheduler/src/Task/Task.php [hunk 1]
```diff
@@ -412,7 +412,7 @@ public function releaseLock(bool $update = true): bool
 
 			if (!in_array($exitCode, [Status::OK, Status::WILL_RESUME]))
 			{
-				$query->set('times_failed = t.times_failed + 1');
+				$query->set('times_failed = t.times_failed');
```
Comment (richard67) [inline administrator/components/com_scheduler/src/Task/Task.php]: Couldn't the complete if block be removed here? The times failed is already set to the same value above at line 404 and bound at 411:

https://github.com/joomla/joomla-cms/blob/78037c2ec72e4ab36a119b6dead08973bd93494d/administrator/components/com_scheduler/src/Task/Task.php#L404

https://github.com/joomla/joomla-cms/blob/78037c2ec72e4ab36a119b6dead08973bd93494d/administrator/components/com_scheduler/src/Task/Task.php#L411

@ditsuke What do you think?

Comment (ditsuke) [reply administrator/components/com_scheduler/src/Task/Task.php]: @richard67 I agree, the block should be removed!

Comment (richard67) [reply administrator/components/com_scheduler/src/Task/Task.php]: @Shubhamverma2796 Could you update your PR in that way? Thanks in advance.

Comment (Shubhamverma2796) [reply administrator/components/com_scheduler/src/Task/Task.php]: Requested changes made.

**Comment by richard67** [comment]
@chmst @pritam825 Could you tested again with the latest changeable this PR? Thanks in advance.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on cce013a2105686d873ca53acfe4f7700a6354bb8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36868">issues.joomla.org/tracker/joomla-cms/36868</a>.</sub>

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on cce013a2105686d873ca53acfe4f7700a6354bb8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36868">issues.joomla.org/tracker/joomla-cms/36868</a>.</sub>

**Comment by RickR2H** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36868">issues.joomla.org/tracker/joomla-cms/36868</a>.</sub>

**Comment by richard67** [comment]
@Shubhamverma2796 There is no need to update your branch as long as there are no conflicts shown. Updating the branch invalidates the human tests counter so I have to add back the tests.

**Comment by richard67** [comment]
Added back the human tests.

**Comment by Shubhamverma2796** [comment]
> @Shubhamverma2796 There is no need to update your branch as long as there are no conflicts shown. Updating the branch invalidates the human tests counter so I have to add back the tests.

Sorry I was not aware of this fact.

**Comment by pritam825** [comment]
I have tested this item :white_check_mark: successfully on 5c80d64298e733aae7b6045beacb1ac4e9dad020<br /><br />working as expected<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36868">issues.joomla.org/tracker/joomla-cms/36868</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**